networkit/cpp/distance/test/SSSPGTest.cpp

```diff
@@ -52,6 +51,36 @@ TEST_F(SSSPGTest, testDijkstra) {
         EXPECT_LE(sssp.distance(stack[i]), sssp.distance(stack[i + 1]));
 }
 
+TEST_F(SSSPGTest, testSSSPThrowsInvalidSource) {
+    constexpr int n = 5;
+    Graph G(n, true);
+    int source = 1;
+    BFS bfs(G, source);
+    int invalid_source = 6;
```

Comment:
All these `int` variables are not needed, just inline the values in the constructors or functions.